src/Colors.ts

```diff
@@ -118,4 +118,39 @@ const interpolateColors = (animationValue: Animated.Adaptable<number>, inputRang
   return colorHSV(h, s, v);
 };
 
+const interpolateColorsRGB = (animationValue: Animated.Adaptable<number>, inputRange: number[], colors: RGBColor[]) => {
+  const r = round(
+    interpolate(animationValue, {
+      inputRange,
+      outputRange: colors.map(c => c.r),
+      extrapolate: Extrapolate.CLAMP,
+    }),
+  );
+  const g = round(
+    interpolate(animationValue, {
+      inputRange,
+      outputRange: colors.map(c => c.g),
+      extrapolate: Extrapolate.CLAMP,
+    }),
+  );
+  const b = round(
+    interpolate(animationValue, {
+      inputRange,
+      outputRange: colors.map(c => c.b),
+      extrapolate: Extrapolate.CLAMP,
+    }),
+  );
+  return color(r, g, b);
+};
+
+const interpolateColors = (
+  animationValue: Animated.Adaptable<number>,
+  inputRange: number[],
+  colors: RGBColor[],
+  colorSpace: "hsv" | "rgb" = "rgb",
```

Comment:
@marcusstenbeck @dsznajder  Shouldn't the default be hsv here to avoid a breaking change?

Reply:
Yeah, I did make the change, but I just might have messed up when committing that. 😨